pythonforandroid/bootstraps/sdl2/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -59,8 +59,11 @@
                  android:icon="@drawable/icon"
                  android:allowBackup="{{ args.allow_backup }}"
                  {% if args.backup_rules %}android:fullBackupContent="@xml/{{ args.backup_rules }}"{% endif %}
+                 {% if args.network_security_config %}android:networkSecurityConfig="{{ args.network_security_config }}"{% endif %}
```

Comment:
I'm wondering if it would be better to have a more generic "additional-application-xml" option by which these new arguments can be added, rather than a new argument for each option. This provides better support for people wanting to use new different arguments in the future, and doesn't clutter up the arguments list so much. What do you think?

Reply:
fully agree, @inclement ! 
for example: https://github.com/kivy/python-for-android/pull/2334/files

What if "additional-application-xml" input parameter would be a file path with an extra XML code to be inserted inside `<application ... >` tag?    because I feel like passing a raw XML code inside the parameter is not a good idea...   what do  you think?

Reply:
okay... as we discussed in Discord chat I will add a normal string parameter `--extra-manifest-application-arguments` that will be injecting value : `<application {{ args.extra_manifest_application_arguments }} >